logstash-core/lib/logstash/environment.rb

```diff
@@ -40,8 +40,18 @@ module Environment
             Setting::String.new("http.host", "127.0.0.1"),
             Setting::PortRange.new("http.port", 9600..9700),
             Setting::String.new("http.environment", "production"),
+            Setting::String.new("queue.type", "persisted", true, ["persisted", "memory", "synchronous"]),
+            Setting::Numeric.new("queue.page_capacity", 250 * 1024 * 1024),
+            Setting::Numeric.new("queue.queue_capacity", 250 * 1024 * 1024)
```

Comment:
and here we should remove the queue.queue_capacity setting because currently we only support a maximum number of events capacity and capping the queue at `250 * 1024 * 1024` events does not make sense.